ftest.sh

```diff
@@ -238,7 +239,7 @@ rm -f core.* *_results.xml
 
 # now run it!
 export PYTHONPATH=./util:../../utils/py/:./util/apricot
-if ! ./launch.py -s \"$TEST_TAG\"; then
+if ! ./launch.py -s $TEST_TAG; then
```

Comment:
This will raise a [shellcheck](https://www.shellcheck.net/) [SC2086: Double quote to prevent globbing and word splitting](https://github.com/koalaman/shellcheck/wiki/SC2086) error at some point in the future when our shellcheck version catches up to the latest.

It is a bit of a pain to use a string that can contain spaces as multiple arguments to a command, but by not following one of the techniques at the above page to deal with it, you risk unintended globbing and word splitting.  This is particularly important given that this user input.

Reply:
Thanks for the feedback.  Fix applied.